src/wmnds/components/loader/_properties.njk.json

```diff
@@ -0,0 +1,9 @@
+{
+  "wmndsLoaderProps": [
+    {
+      "name": "size",
+      "type": "boolean",
+      "description": "Determines the display size of the loader. If left blank, the loader will display at the default size. Other accepted values are <code class='wmnds-website-inline-code'>small</code> or <code class='wmnds-website-inline-code'>large</code>."
+    }
```

Comment:
The component also has a "customClass" paramater. That should also be displayed in the component example and documentation.

Also, change this param name from "customClass" to "classes" see how link/button is done on the release branch.